linchpin/utils/dataparser.py

```diff
@@ -63,8 +63,14 @@ def process(self, file_w_path, data_w_path=None):
                 except Exception:
```

Comment:
Isn't this the real issue?

Reply:
Per a conversation in IRC between @p3ck and myself, we've agreed to rewrite this function (and a few other adjustments) to deal with data input a little better. The convention commonly used is to put an @ when passing in file data. We'll adjust the input and the code to parse this data with that expectation.